test/unit/adapters/http.js

```diff
@@ -303,6 +303,64 @@ describe('supports http with nodejs', function () {
     });
   });
 
```

Comment:
why no responseType json test? https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType

Reply:
Thanks for review. I miss it and i will add test for it later.